TMA.py.lprof

```diff
@@ -0,0 +1,3 @@
+�c_line_profiler
+LineStats
+q)�q}q(UunitqG>����Utimingsq}qU�/Users/lcoop22/anaconda/lib/python2.7/site-packages/histomicstk-0.1.0-py2.7.egg/histomicstk/segmentation/nuclear/MaxClustering.pyqKU
MaxClusteringq�]sub.
```

Comment:
This is some temp file. Add .lprof to .gitignore. Then github will ignore it from commits.